nonCisMaleInCs/DetailsView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {StyleSheet, View, Text, Image, Button, Alert, TouchableOpacity} from 'react-native';
+// import { Icon } from 'react-native-elements' 
```

Comment:
please remove these lines if we are not using....

Reply:
Done